### PR TITLE
Create CD action

### PR DESCRIPTION
**Pull request by WalshyDev**
You've used actions much more than me so let me know if there's anything you spot which is not correct.

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    if: "! contains(toJSON(github.event.commits.*.msg), '[ci skip]')"
```
Comment (TheBusyBiscuit) [inline .github/workflows/maven.yml]: Iirc the exclamation mark is still broken so you may need to do == false here

**Comment by WalshyDev** [comment]
Know of a good action to change the version in the pom.xml?

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
```
Comment (TheBusyBiscuit) [inline .github/workflows/maven.yml]: Why the "--file pom.xml" here?

File: .github/workflows/maven.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - name: Create a Release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: $GITHUB_RUN_NUMBER
+        release_name: Build $GITHUB_RUN_NUMBER
```
Comment (TheBusyBiscuit) [inline .github/workflows/maven.yml]: Would add a hashtag before the number for aesthetic reasons

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by TheBusyBiscuit** [comment]
> Know of a good action to change the version in the pom.xml?

Well, for my builds page I use my xml-library and just change the version with that, could theoretically pack this into an action.

File: .github/workflows/maven.yml (hunk 1 above)
Comment (WalshyDev) [reply .github/workflows/maven.yml]: Is there an issue ticket for this somewhere?

**Comment by WalshyDev** [comment]
> > Know of a good action to change the version in the pom.xml?
> 
> Well, for my builds page I use my xml-library and just change the version with that, could theoretically pack this into an action.

If you could that would be nice :heart:

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**